GNUmakefile

```diff
@@ -45,8 +45,7 @@ format:
 generate:
 	@echo "--> Running go generate"
 	@go generate $(PACKAGES)
-	@sed -e 's|github.com/hashicorp/nomad/vendor/github.com/ugorji/go/codec|github.com/ugorji/go/codec|' nomad/structs/structs.generated.go >> structs.gen.tmp
-	@mv structs.gen.tmp nomad/structs/structs.generated.go
+	@sed -i -e 's|github.com/hashicorp/nomad/vendor/github.com/ugorji/go/codec|github.com/ugorji/go/codec|' nomad/structs/structs.generated.go
```

Comment:
There was a reason we didn't use the inlace... I think that it doesn't work on Mac